src/runtime/internal/dom.ts

```diff
@@ -163,7 +163,8 @@ export function append_hydration(target: NodeEx, node: NodeEx) {
 	if (is_hydrating) {
 		init_hydrate(target);
 
-		if ((target.actual_end_child === undefined) || ((target.actual_end_child !== null) && (target.actual_end_child.parentElement !== target))) {
+		// Note: It is important to use `Node.parentNode` instead `Node.parentElement` for a simple legacy browsers compatibility.
```

Comment:
I'm not sure I'd include the comment because it seems like if we want to have it then it should be everywhere that we use `parentNode` and that would start to get really noisy
```suggestion
```

Reply:
> 

Oh it's a matter of a taste, I just wanted no one to think (almost correctly) in the future that "parentElement" is more correct here.
However, I accept this convention.